### ISSUE TITLE
[Crash] Random crash (OpenGL API ERROR: 1282)

### ISSUE THREAD

**Issue by juliocoliveira** [sp614x/optifine#6476]
## Description of Issue
I'm getting random crashes after 2-5 minutes after connecting to a server, I saw a lot of optifine in the errors so i thought it is caused by optifine
I'm using Misa 64 Texture Pack.

## Steps to Reproduce
Install forge+Optifine or Fabric+OptiFabric+Optifine and enable Render Regions.

## OptiFine Version
OptiFine HD U H4 (1.18.1)

## Installation Method
Jar into mods folder

## Fabric/Forge Version
39.0.5

## Other Installed Mods
AdvancedCompas-forge-1.18-1.2.6.jar
CraftPresence-Forge-1.18.1-Release-1.8.8.jar
 harvest-1.18.1-1.1.jar
 jei-1.18.1-9.1.0.41.jar
 OptiFine_1.18.1_HD_U_H4.jar
 voicechat-forge-1.18.1-2.1.22.jar
wthit-forge-4.3.1.jar
XaerosWorldMap_1.18.6.1_Forge_1.18.jar
 Xaeros_Minimap_21.22.3.1_Forge_1.18.jar

## Log Files/Crash Reports
(https://paste.ee/p/kPw3g)

## F3 Debug Screenshot
![Untitled-1](https://user-images.githubusercontent.com/6606725/145908021-2727c0d0-c0f3-4945-8871-850713dd5667.jpg)

## Prior Testing
 No

**Comment by juliocoliveira** [comment]
I also tried with the latest OptiFabric for 1.18.1, The issue persists but does not crash the game, only generates the error in the chat window, and bugs out ShadowPlay (Half of the screen is black), had to logout, login on windows.

The issue seems to be the same of #2548.
I saw the tag "Render Regions" and went into Optifine options and disabled it.
Since then never crashed anymore, over one hour of gameplay, and walked over 8k blocks, no crash what so ever.

It seems to be an old bug with Render Regions, would be cool if it got fixed, Still didn't see any difference in performance with it disabled.
